dot_config/nixpkgs/flake.nix

```diff
@@ -0,0 +1,84 @@
+{
+  description = "Home-manager configuration";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-21.11";
+    nixpkgs-unstable.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+
+    neovim-nightly-overlay.url = "github:nix-community/neovim-nightly-overlay";
+
+    nixgl = {
+      url = "github:guibou/nixGL";
+      flake = false;
+    };
+
+    home-manager = {
+      url = "github:nix-community/home-manager/release-21.11";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, ... }@inputs:
+    let
+      overlay-unstable = final: prev: {
+        unstable = import inputs.nixpkgs-unstable {
+          system = prev.system;
+          config = {
+            allowUnfree = true;
+          };
+        };
+      };
+      overlay-nixgl = final: prev: {
+        nixgl = import inputs.nixgl {
+          pkgs = inputs.nixpkgs.legacyPackages.${prev.system};
```

Comment:
nixgl is now using nixpkgs stable whereas before we were using the unstable channel - don't think there was a reason for using unstable and perhaps stable will be more... stable